src/rpc/blockchain.cpp

```diff
@@ -2219,7 +2223,10 @@ static RPCHelpMan savemempool()
         throw JSONRPCError(RPC_MISC_ERROR, "Unable to dump mempool to disk");
     }
 
-    return NullUniValue;
+    UniValue ret(UniValue::VOBJ);
+    ret.pushKV("filename", (gArgs.GetDataDirNet() / "mempool.dat").string());
```

Comment:
Would be nice to use `request.context.node.args` and not the global.

Reply:
Done.